recipes/jupyterlab-plugin-graph/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "jupyterlab-plugin-graph" %}
+{% set version = "0.2.1" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 9131fa511e2219aa502e7af86ae5e86dd07d64ec91d33188c910df0ce490ccfa
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+    - jupyter-packaging >=0.10,<1
+  run:
+    - jupyterlab ~=3.0
+    - python >=3.6
+
+test:
+  commands:
+    - pip check
+    - test -f "${PREFIX}/share/jupyter/labextensions/jupyterlab-plugin-graph/package.json"                                    # [unix]
+    - if exist %PREFIX%\\share\\jupyter\\labextensions\\jupyterlab-plugin-graph\\package.json (exit 0) else (exit 1)          # [win]
+  requires:
+    - pip
+
+about:
+  home: https://github.com/jupyterlab-contrib/jupyterlab-plugin-graph
+  summary: JupyterLab extension to show an interactive dependency graph of the installed plugins 
+  license: BSD-3-Clause
+  license_file: LICENSE
+
+extra:
+  recipe-maintainers:
+    - jtpio
+    - fcollonval
```

Comment:
cc @fcollonval if you agree to be listed here

Reply:
I agree to be added as maintainer.

Reply:
Thanks!